bin/apalache-mc

```diff
@@ -38,7 +38,11 @@ TLA_LIB="$DIR/src/tla"
 # This is a temporary solution for handling TLA_PATH via a Java variable.
 # See apalache/#187.
 # Once tlaplus/#490,#493 are released, remove this workaround.
-(test -z "$TLA_PATH" && TLA_PATH="$TLA_LIB") || TLA_PATH="$TLA_LIB:$TLA_PATH"
```

Comment:
I found that these two pieces of code are actually not equivalent. The expression in parentheses spawns a new shell, sets a variable and then terminates the shell. Effectively, it never updates `TLA_PATH`. That is why I introduced the simple `if-else-fi`.

Reply:
ah! The subshell thing! Good catch. Thank you for explaining :)